### ISSUE TITLE
JEI multi block preview not working correctly

### ISSUE THREAD

**Issue by XXxinsanityxX** [GregTechCE/GregTech#945]
The JEI multi block preview category is only showing the casings of the multi blocks its not showing the controllers or the buses/hatches the multi might use.

Mod-list:
CodeChickenLib-1.12.2-3.2.2.353-universal
CraftingTweaks_1.12.2-8.1.9
CraftTweaker2-1.12-4.1.17
FastLeafDecay-v14
foamfix-0.10.3-1.12.2
gregtech-1.12.2-1.8.5.436
inventorysorter-1.12.2-1.13.3+57
jei_1.12.2-4.15.0.268
journeymap-1.12.2-5.5.4
modtweaker-4.0.17
MouseTweaks-2.10-mc1.12.2
MTLib-3.0.6
phosphor-1.12.2-0.2.6+build50-universal
theoneprobe-1.12-1.4.28
VanillaFix-1.0.10-99
weirdinggadget-1.12.2-2.1.16-universal

**Comment by UnlimatedStone9** [comment]
I assume this is what they're talking about.
I only have GTCE installed with JEI and this occurs
![image](https://user-images.githubusercontent.com/40124890/63599973-95d71300-c5ba-11e9-834e-62cf3645b311.png)

**Comment by IcedReaper** [comment]
This seems to be a problem with every version above 1.8.4.419.
When I had most recent version installed I saw the rendering errors and I couldn't see any recipe that a GTCE machine had saved.
I checked back the versions as I knew it was working for me a week or so ago.
I gone back to the version 1.8.4.419 and it worked.
I tried the version 1.8.4.432 and it had the same errors again.

So I think it's something that changed in this version.
